net/adguardhome/files/adguardhome.init

```diff
@@ -4,10 +4,9 @@ PROG=/usr/bin/AdGuardHome
 
 USE_PROCD=1
 
-# starts after network starts
-START=21
-# stops before networking stops
-STOP=89
+# Matches values used by the official AdGuardHome procd init script
+START=95
```

Comment:
95 might be a bit late, since this provides the basic service of DNS.

From the forum post it seems that starting right after network doesn't work because network might still be busy bringing up interfaces so AGH couldn't attach to them (in case you use a specific subnet like `192.168.1.1` instead of `0.0.0.0`).

Maybe delaying until 50 is a bit better here to ensure that network fully started but not too late that other network-related services have trouble working (9x range seems to be filled with applications from my cursory look at this repo). This number is taken from [uhttpd](https://github.com/openwrt/openwrt/blob/8c1d5129ee8464b3a240d73c48eb2d166537c1a8/package/network/services/uhttpd/files/uhttpd.init#L4) of which I assume also have the requirement of waiting for network to fully activate.

Reply:
That seems reasonable. It seems to be that AdGuard Home can't start just after network, likely as you say due to interfaces not being available potentially.